tutorials/plot_stats.py

```diff
@@ -11,40 +11,126 @@
 # Applying statistical measures to spiking data
 # ---------------------------------------------
 #
-# Words, words, words.
+# Sections
+# ~~~~~~~~
+#
+# This tutorial contains the following sections:
+#
+# 1. Compute and plot different shuffles of spikes
+# 2. Compute empirical p-value from distribution of spikes
+# 3. Compute z-score from distribution of surrogates
 #
 
 ###################################################################################################
 
 # sphinx_gallery_thumbnail_number = 1
 
-import spiketools
+# import auxiliary libraries
+import numpy as np
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+# import all functions from spiketools.stats
+from spiketools.stats.generators import *
+from spiketools.stats.permutations import *
+from spiketools.stats.shuffle import *
 
 ###################################################################################################
 
-# Settings
-...
+# Generate spike times for spikes at 10Hz for 1000 seconds
+poisson_generator = poisson_train(10, 1000)
+# get spike in seconds
+spikes_s = np.array([spike for spike in poisson_generator])
+# convert to milliseconds
+spikes_ms = spikes_s*1000
+spikes_ms = np.unique(spikes_ms.astype(int))
 
 ###################################################################################################
-# Subtitle
+# 1. Compute and plot different shuffles of spikes
 # ~~~~~~~~
```

Comment:
The header underlines (~~~), should be the same length of the title. 

Something like:
```
This is a title
~~~~~~~~~
```